projects/plugins/social/src/js/components/stat-cards/styles.module.scss

```diff
@@ -17,7 +19,11 @@
 	box-shadow: 0px 0px 40px rgba( 0, 0, 0, 0.08 );
 	border-radius: 4px;
 	padding: calc( var( --spacing-base ) * 3 );
-	max-width: 170px;
+	width: calc( 100% - var( --spacing-base ) * 2 / 2 );
```

Comment:
You can also use Flexbox to avoid calculating the width:

```css
flex: 0 1 auto;
width: 100%;
```

Reply:
Yeah good point!